### PR TITLE
feaT: Add Loki egg

### PR DESCRIPTION
**Pull request by realkarmakun**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?

### New Server Submissions:

1. [x] Does your submission pass tests (server is connectable)?
3. [x] Have you added the server to the main README.md?
4. [x] Have you added a unique README.md for the server you are adding?

Closes https://github.com/parkervcp/eggs/issues/1590

### CODE REVIEW

**Comment by DatMayo** [comment]
You can add "closes #1590" to your PR, so it will be automaticly closed when your PR is merged :)

**Comment by realkarmakun** [comment]
> You can add "closes #1590" to your PR, so it will be automaticly closed when your PR is merged :)

I'm sorry, where should I do that exactly? Right here in the comments? Or in commit message? I don't quite understand what you want..

**Comment by realkarmakun** [comment]
closes https://github.com/parkervcp/eggs/issues/1590

**Comment by realkarmakun** [comment]
This closes https://github.com/parkervcp/eggs/issues/1590

**Comment by realkarmakun** [comment]
@DatMayo Is this what you wanted? To change the PR's name?

**Review by Software-Noob — CHANGES_REQUESTED** [review]

File: monitoring/loki/egg-loki.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2022-03-23T19:24:39+03:00",
+    "name": "Loki",
+    "author": "unknown@unknown.com",
+    "description": "Prometheus but for logs. This egg is for Loki instance only! You usually need pushing agents like Promtail to put logs inside",
+    "features": null,
+    "images": [
+        "quay.io\/parkervcp\/pterodactyl-images:base_debian"
```
[the remark this answers is not in the document]
Comment (realkarmakun) [reply monitoring/loki/egg-loki.json]: Done in https://github.com/parkervcp/eggs/pull/1606/commits/33a72c0069d503047d57a30d61d74d6b2114987a

**Review by realkarmakun — COMMENTED** [review]
Using new yolk repo now https://github.com/parkervcp/eggs/pull/1606/commits/33a72c0069d503047d57a30d61d74d6b2114987a

**Review by Software-Noob — APPROVED** [review]
LGTM. Thank you for contributing :+1:

I've cleaned it up a tad and fixed the readme.  Using an existing install image that contains the necessary packages speeds up the installation. Globbing is already enabled in Debian, and creating a new variable just to assign an existing variable value to it is unnecessary. 

Preferably we always echo what we're doing, so it's easier to troubleshoot once users report issues. I've added an echo of the curl URL in use.

**Pull request merged**